content/en/blog/getting-started/index.md

```diff
@@ -0,0 +1,252 @@
+service agent development without using any extra proxies or network tools.
+
+*"So, I have the agency up and running. What next?"*
+
+**TL;DR** Take a look at the examples found in [the sample repository](https://github.com/findy-network/identity-hackathon-2023)!
+
+## Run the CLI
+
+["findy-agent-cli"](https://github.com/findy-network/findy-agent-cli)
+is a command-line tool that provides all the required
+agent manipulation functionality. It provides means to quickly test out
+the issuing and verifying *before writing any code*.
+
+[The sample script](https://github.com/findy-network/identity-hackathon-2023/tree/master/cli#cli-example)
+is a good starting point. It shows how to allocate an agent in the cloud
+and issue and verify credentials using a simple chatbot. You can run it by cloning [the repository](https://github.com/findy-network/identity-hackathon-2023)
+and following the instructions in the README.
+
+### CLI Script Initialization Phase
+
+The sample script initialization phase allocates a new agent from the agency (**1**)
+and authenticates the CLI user (**2-3**). The authentication returns a JWT token
+exposed to the script environment so that further CLI calls can utilize it
+automatically.
+
+For the agent to issue credentials, an applicable schema needs to exist.
+The schema describes the contents of a credential, i.e., which attributes the credential contains.
+The sample script creates a schema "foobar" with a single attribute "foo" (**4-5**).
+
+There needs to be more than the mere schema for the issuing process; the agent needs also to create
+and publish its credential definition (**6-7**) attached to the created schema so that it can issue
+credentials and verifiers can verify the proof presentations against the published credential definition.
+
+We assume that the holder operates a web wallet and has taken it into use (**8-9**).
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Register
+    CLI->>Agency: Login
+    Agency-->>CLI: JWT token
+    CLI->>Agency: Create schema
+    Agency-->>CLI: Schema id
+    CLI->>Agency: Create cred def
+    Agency-->>CLI: Cred def id
+    Note left of Web Wallet: Holder takes the web wallet in use
+    Web Wallet->>Agency: Register
+    Web Wallet->>Agency: Login
+```
+
+### CLI Script Issuing Credential
+
+The next task is to create a pairwise connection between the agent operated by the CLI user
+and the web wallet user. The pairwise connection is an encrypted pipe between the two
+agents that they can use to exchange data securely. The CLI script creates an invitation (**1-2**)
+and prints it out (**3**) as a QR code that the web wallet user can read (**5**).
+
+Once the new connection id is known, the CLI script starts a chatbot (**4**) for the new connection.
+The bot logic follows the rules for changing the bot states in the YAML configuration file. Therefore,
```

Comment:
YAML configuration file -> YAML file?
or YAML state machine file. The exact syntax will be published as a blog post soon.

Or something similar...